ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/AEMDownloadManger.java

```diff
@@ -455,6 +470,8 @@ private File streamResource(@NonNull final InputStream in) throws IOException {
         volatile boolean mStarted = false;
         volatile boolean mForce = false;
 
+        final SettableFuture<Boolean> mResult = SettableFuture.create();
```

Comment:
the only reason I made this a `Boolean` Future is in case we wanted to report whether a task actually did something or not.